benefit/data/sheets/CaseType/CaseType.json

```diff
@@ -1 +1 @@
-[{"LiveFrom": "01/01/2018", "ID": "Benefit", "Name": "SSCS Case v5.15.0_${CCD_DEF_E}", "Description": "SSCS Benefit Cases", "JurisdictionID": "SSCS", "PrintableDocumentsUrl": "http://ccd-data-store-api-prod.service.core-compute-prod.internal/callback/jurisdictions/SSCS/case-types/Benefit/documents", "RetriesTimeoutURLPrintEvent": "5,10,15", "SecurityClassification": "PUBLIC"}]
+[{"LiveFrom": "01/01/2018", "ID": "Benefit", "Name": "SSCS Case v5.15.0_${CCD_DEF_E}", "Description": "SSCS Benefit Cases", "JurisdictionID": "SSCS", "PrintableDocumentsUrl": "http://ccd-data-store-api-prod.service.core-compute-prod.internal/callback/jurisdictions/SSCS/case-types/Benefit/documents", "RetriesTimeoutURLPrintEvent": "5,10,15", "SecurityClassification": "PUBLIC"}]
```

Comment:
After discussion, this should be a nonprod change, so this change will be reverted and applied to the relevant nonprod sheet

Reply:
Relevant changes applied. Resolving this conversation.